### PR TITLE
fix: add ls part for iac ignore button [IDE-683]

### PR DESCRIPTION
**Pull request by DariusZdroba**
### Description

This is the logic needed for adding the IaC ignore button.
It has the HTML which has the button, hidden by default, enabled in **IntelliJ**.

A `FilePath` was also added to the TemplateData which is needed to trigger the IgnoreService for the specific issue, and the before named 'Path' variable was renamed to `ResourcePath` which is a more accurate name for what it represents.

A file path was also added to the template for enabling to execute the ignore issue comand which also needs file path. 
### Checklist

- [x] Tests added and all succeed
- [x] Linted
- [ ] README.md updated, if user-facing
- [ ] License file updated, if new 3rd-party dependency is introduced

### CODE REVIEW

File: infrastructure/iac/iac_html.go [hunk 1]
```diff
@@ -39,7 +39,8 @@ type TemplateData struct {
 	SeverityIcon template.HTML
 	Description  template.HTML
 	Remediation  template.HTML
-	Path         template.HTML
+	ResourcePath template.HTML
```
Comment (DariusZdroba) [inline infrastructure/iac/iac_html.go:42]: The "Path" name is a bit ambiguous, in this case the Path refers to where the issue can be found, example: 
"[DocId: 0] > input > spec > template > spec > hostNetwork" .

File: infrastructure/iac/template/index.html [hunk 2]
```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
       {{end}}
     </div>
   </section>
+  <!-- Ignore In Line Actions -->
```
Comment (bastiandoetsch) [inline infrastructure/iac/template/index.html:86]: nitpick: this comment is probably not accurate anymore

File: infrastructure/iac/template/index.html [hunk 3]
```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
...
+  {{if not .Issue.IsIgnored}}
```
Comment (bastiandoetsch) [inline infrastructure/iac/template/index.html:87]: comment: strictly speaking this is not necessary, but nice for later, when ignores are provided from API/Findings model.

File: infrastructure/iac/template/index.html [hunk 4]
```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
...
+  <footer id="ignore-container" class="ignore-action-container hidden">
+    <div id="ignore-actions">
+      <div class="actions row">
+        <button id="ignore-file-issue" issue="{{.Issue.ID}}" path="{{.ResourcePath}}" filePath="{{.FilePath}}" class="ignore-button secondary">Ignore in this file</button>
```
Comment (bastiandoetsch) [inline infrastructure/iac/template/index.html:91]: nitpick: why not rename `path` here, too?

File: infrastructure/iac/template/index.html [hunk 5]
```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
...
+      </div>
+    </div>
+  </footer>
+  {{end}}
```
Comment (bastiandoetsch) [inline infrastructure/iac/template/index.html:95]: question: what is `end`ed here?

**Review by bastiandoetsch — APPROVED** [review]

**Comment by bastiandoetsch** [comment]
Consider testing that the template is filled correctly for the ignores stuff ;)

File: infrastructure/iac/template/index.html (hunk 5 above)
Comment (DariusZdroba) [reply infrastructure/iac/template/index.html:95]: That is the end of the {{if not .Issue.IsIgnored}} statement.

**Pull request merged**